docker-compose-local.yml

```diff
@@ -5,13 +5,10 @@ services:
     container_name: simulator-server
     environment:
       - PORT=1212
-      - KUBE_SCHEDULER_SIMULATOR_ETCD_URL=http://simulator-etcd:2379
-      - CORS_ALLOWED_ORIGIN_LIST=http://${SIMULATOR_EXTERNAL_IP:-localhost}:3000
-      - KUBE_API_HOST=0.0.0.0
-      - KUBE_API_PORT=3131
+      - KUBE_SCHEDULER_SIMULATOR_ETCD_URL=http://simulator-cluster:2379
```

Comment:
It sounds simple and clear for me to define as ETCD_URL

Reply:
We don't have to worry about the env. The configuration thru env is deprecated and we'll remove them eventually. So, the current way of configuring is basically thru scheduler config file, which has EtcdURL instead of this messy long title.